src/components/teasers/teaser-grid.tsx

```diff
@@ -63,14 +63,15 @@ const StyledTeaserGrid = styled.div<{ amountOfChildren: number }>`
 `;
 
 interface TeaserGridProps {
+  className?: string;
   children: React.ReactNode | React.ReactNode[];
 }
 
 export const TeaserGrid = (props: TeaserGridProps): JSX.Element => {
   const amountOfChildren = React.Children.count(props.children);
 
   return (
-    <ScrollContainer>
+    <ScrollContainer className={props.className}>
```

Comment:
I actively use an updated teaser grid component hence the support for styled component here